Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
I don't like the **make everyhing final** concept. Sometime the dev wants to override existing functions.